crates/bevy_ui/src/render/ui_vertex_output.wgsl

```diff
@@ -5,7 +5,9 @@ struct UiVertexOutput {
     @location(0) uv: vec2<f32>,
     // The size of the borders in UV space. Order is Left, Right, Top, Bottom.
     @location(1) border_widths: vec4<f32>,
+    // The size of the borders in pixels. Order is top left, top right, bottom right, bottom left.
+    @location(2) border_radius: vec4<f32>,
     // The size of the node in pixels. Order is width, height.
-    @location(2) @interpolate(flat) size: vec2<f32>,
+    @location(3) @interpolate(flat) size: vec2<f32>,
```

Comment:
Can / should we avoid swapping this by moving the new `border_radius` binding to slot 3 rather than displacing `size`?